transcript_tools/style_guide.md

```diff
@@ -0,0 +1,132 @@
+- Filler words: "The answer is, uh, hm, six."
+- False starts: "The answer should... the answer is six."
+- Repeated words: "The answer is, is six."
+All of these can be written "The answer is six."
+
+Often the word "like" can be a filler word: "The, like, answer, like, is six."
+It probably improves readability to remove it.  Be careful, though— sometimes
+"like" is part of the meaning of the sentence, e.g. "They were like, this is
+broken." (Where "like" indicates a thought or statement attributed to someone
+else.)
+
+How much to trim is ultimately a judgment call.  Try to leave the speaker's
+style in place but also make the sentences readable.
+
+Try to insert sentence breaks where needed; one sentence that goes on for too
+long hurts readability.  If that means you need to start sentences with "And",
+"But", "So", etc., that's still better than run-on sentences that go on forever.
+
+If a speaker cuts themselves off mid-sentence, and jumps topics, or interrupts
+or corrects themselves, use the UTF-8 em-dash character "—" to mark the break.
+Example: "As you would expect, you can compile— the Rust compiler will use the
+usual algorithms."
+
+Also use an em-dash "—" to mark the point when a speaker stops suddenly (and
+doesn't resume) or if a speaker is interrupted by someone else.
+
+Don't try to fix sentences that are not grammatically correct.  If that's what
+the speaker said, it's what should go in the transcript.  If it's not clear
+whether they used the correct word (e.g. you can't tell whether they used the
+right verb tense), by all means use the correct one by default.
+
+## Hyperlinks
+
+So far the transcripts have not contained many hyperlinks.  This hasn't been
+discussed among contributors yet.
+
+What do you think the link policy should be?
+
+## When to use `code` tags
+
+Use the inline `code` style when it's clearly a Rust symbol, trait, type, macro,
+etc. that's being referred to, e.g. `Option`, `Result`, `TryFrom`, etc.  Editor
+search/replace isn't a good idea, because a speaker may talk about a result and
+a `Result` in the same paragraph— only use `code` when it's referring to the
+specific Rust element.
+
+It's fine (and usually helps readability) to translate certain things to Rust
+syntax. For example, if someone says "vec of reference to str", it's OK to write
+`Vec<&str>`, as long as it's clear that's what they meant.
+
+It's usually much more readable to translate to `code` when the speaker is
+spelling out Rust syntax character by character. I.e. if the speaker says
+"bracket A comma B comma C", it's best to replace that with `[a, b, c]`.
+
+When a macro is put in a `code` tag, add the exclamation point, e.g. `println!`
+
+Use `code` when a crate is being named, e.g. `serde`.  If the crate name has a
+dash, then it's probably wise to use the dashed name when referring to the crate
+(e.g. `num-traits`) but the underscored name when referring to it in Rust code,
```

Comment:
If we want to be _really_ diligent here, we'd look up whether the crate on crates.io uses `-` or `_`.

Reply:
That's what I meant-- I can clarify the text.

Reply:
Isn't that what it says already?  "If the crate name has a dash, then..."

Reply:
Ah, I read this as "if the crate name (as read sounds like it) has a dash", rather than "if the crate name (when you look it up on crates.io) has a dash".